Added tooltip to map to show summary statistics

Forsythia-olive opened the pull request:
Added tooltip to map to show summary statistics

Forsythia-olive committed:
Added a tooltip on map for total financial loss

Forsythia-olive committed:
Update tooltip to reflect the currency in Billions

Forsythia-olive committed:
Merge branch 'main' into map-tooltip

fishydays reviewed (changes requested):
Hi @Forsythia-olive, thanks for all the hard work! The changes look great overall, just a few small things that I think we should change to keep the dashboard a bit cleaner and more consistent: 1) Please remove the title from the map, it is redundant information as we now have titles for all graphs built in the layout already. 2) To keep things consistent, I think it is best to label the "Assessed Value" to be "Economic Loss" instead, that way users will know that it is the same category as the summary card and the time series graph. 3) It will be better if we do not show the index when we hover over the map, it can be a bit misleading right now

Forsythia-olive committed:
Removed title and corrected the assessed value to economic value

Forsythia-olive committed:
Merge branch 'map-tooltip' of github.com:UBC-MDS/DSCI-532_2025_27_CA_Wildfire-Dashboard into map-tooltip

Forsythia-olive commented:
> Hi @Forsythia-olive, thanks for all the hard work! The changes look great overall, just a few small things that I think we should change to keep the dashboard a bit cleaner and more consistent: 1) Please remove the title from the map, it is redundant information as we now have titles for all graphs built in the layout already. 2) To keep things consistent, I think it is best to label the "Assessed Value" to be "Economic Loss" instead, that way users will know that it is the same category as the summary card and the time series graph. 3) It will be better if we do not show the index when we hover over the map, it can be a bit misleading right now

I have removed the redundant title and changed Assessed value to Economic Loss. However, since yesterday  I tried resolving the index but it does not work. the approach that worked broke the link between the on-click functionality.  I am still looking into it but am not sure it can be added to this weeks deployment

Forsythia-olive committed:
Modify map tooltip to exclude the index column

fishydays committed:
Merge branch 'main' into map-tooltip

fishydays commented:
> > Hi @Forsythia-olive, thanks for all the hard work! The changes look great overall, just a few small things that I think we should change to keep the dashboard a bit cleaner and more consistent: 1) Please remove the title from the map, it is redundant information as we now have titles for all graphs built in the layout already. 2) To keep things consistent, I think it is best to label the "Assessed Value" to be "Economic Loss" instead, that way users will know that it is the same category as the summary card and the time series graph. 3) It will be better if we do not show the index when we hover over the map, it can be a bit misleading right now
> 
> I have removed the redundant title and changed Assessed value to Economic Loss. However, since yesterday I tried resolving the index but it does not work. the approach that worked broke the link between the on-click functionality. I am still looking into it but am not sure it can be added to this weeks deployment

Ok, thanks for looking into it! let's just mark it as something to look into next week
